### PR TITLE
Fixes Cargo related Alt Title glitch, adds some new ones.

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

Title.

## How This Contributes To The Skyrat Roleplay Experience

Cargo Tech and Shaft Miner were unselectable from the title menu.

More flavour is good, it also allows you to have a title that reflects what you want to do during the round.  Beekeeping, cakes, a farmer gimmick. Etc.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  It works.
</details>

## Changelog

:cl:
add: Multiple titles have been added.
fix: You can select Cargo Tech and Shaft Miner, again.
/:cl:

### CODE REVIEW

**Comment by desertmote** [comment]
Cargo tech and shaft miner being an unselectable job title is an intended feature.

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 1]
```diff
@@ -63,6 +63,7 @@
 		"Bitdomain Technician",
 		"Data Retrieval Specialist",
 		"Netdiver",
+		"Netrunner",
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: I've already denied it once, I'll deny it again. That's a title from Cyberpunk, I'm not interested, as much as I love that game.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
It was technically intended that they couldn't be chosen, I'll have to talk with Jungle to see if we want to keep it that way or not. I wouldn't mind making them selectable again, I think everyone knows Cargo is from the FTU now.

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm (hunk 1 above)
Comment (theselfish) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: Never even knew that. I just thought it was a neat thing.

Comment (theselfish) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: And, that isn't me playing dumb, I've never played nor have any intent to play Cyberrunner, I just thought Netrunner ran off the tongue well.

**Comment by theselfish** [comment]
@GoldenAlpharex Waiting for the worms to come.

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**